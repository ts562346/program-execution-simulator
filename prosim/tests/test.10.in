test1 7
BLOCK 1
LOOP 4
  DOOP 2
  BLOCK 3
END
DOOP 1
HALT
