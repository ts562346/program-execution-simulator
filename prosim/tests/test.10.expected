00000: BLOCK
00001: DOOP
00003: BLOCK
00006: DOOP
00008: BLOCK
00011: DOOP
00013: BLOCK
00016: DOOP
00018: BLOCK
00021: DOOP
00022: HALT

DOOP count : 5
DOOP time  : 9
BLOCK count: 5
BLOCK time : 13
